modules/consequences-of-disabling-telemetry.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * telemetry/opting-out-of-telemetry.adoc
+:_content-type: CONCEPT
+[id="consequences-of-disabling-telemetry_{context}"]
+= Consequences of disabling Telemetry
+
+In {rh-rhacs-first} version 4.0, you can opt out of Telemetry.
+However, Telemetry is embedded as a core component, and therefore, opting out is strongly discouraged.
```

Comment:
**suggestion:** Maybe "However, telemetry is embedded as a core component, so opting out is strongly discouraged.".